CONTRIBUTING.md

```diff
@@ -144,83 +143,101 @@ Ensure you use version 6.4.0 to prevent conflicts. Check the appropriate release
 
 We follow [google coding style](http://google.github.io/styleguide/).
 
-## Steps to commit
+## General Workflow
 
-1. Leave the issues. (If the issue is a straightforward one, it is indeed possible to skip this process.)
-2. Create a new branch whose name starts with an imperative verb above like `feat/implement-xyz`.
-   Branch name should consist of [a-z|0-9|-]. The prefix keyword should be one of followings defined [Commit type](#commit-type)
-3. Make your changes.
-4. Run a formatting tool if you make changes to codes.
-5. Run a lint tool if you make changes to codes.
-6. Run unittests if you make changes to codes.
-7. Rebase your local repository.
+Follow this workflow when working on our repositories:
 
-   ```shell
-   > git fetch origin -p
-   > git rebase origin/dev
-   ```
+1. Select or create an issue.
+    - Note: If the "issue" is straightforward, you do not need to create a new issue.
+
+2. Create a new branch for this issue.
+
+3. On the new branch, make your changes to solve the issue.
+    - If you work on the code...
+        1. Test your code against...
+            - A formatting tool
+            - A lint tool
+            - Unittests
+
+4. Commit your changes following our rules.
```

Comment:
Can you add a link to "our rules" please?

Reply:
Thought this would fit better in a different commit where I added links to sections after I reorganize.